overlays/patches/0001-Fix-console-rootfs-for-imx8qm.patch

```diff
@@ -1,15 +1,15 @@
 diff --git a/Makefile b/Makefile
-index f0f6a4b..e76121e 100644
+index 4588033..cd544f2 100644
 --- a/Makefile
 +++ b/Makefile
 @@ -37,6 +37,7 @@ FILES = \
-	etc/s6-linux-init/run-image/service/getty-tty3/run \
-	etc/s6-linux-init/run-image/service/getty-tty4/run \
-	etc/s6-linux-init/run-image/service/getty-ttyS0/run \
+ 	etc/s6-linux-init/run-image/service/getty-tty3/run \
+ 	etc/s6-linux-init/run-image/service/getty-tty4/run \
+ 	etc/s6-linux-init/run-image/service/getty-ttyS0/run \
```

Comment:
These spaces are added by git itself. I believe it is OK to merge, nevertheless git will warn about whitespaces. It is a known git behavior.